src/client/components/ModalViewCancelEvent/ModalViewCancelEvent.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './ModalViewCancelEvent.styles.css';
+
+export const ModalCancelEvent = (props) => {
+  return (
+    <div className="modal-cancel-event-box">
+      <div className="modal-cancel-event-close-x">
+        <span>x</span>
+      </div>
+      <div className="modal-cancel-event-main">
+        <div className="modal-cancel-event-heading">{props.heading}</div>
+        <div className="modal-cancel-event-text">{props.text}</div>
+        <button className="modal-cancel-event-button" type="submit">
+          {props.buttonText}
+        </button>
+      </div>
+    </div>
```

Comment:
You did way too much - we have a `Modal` component that already simplifies the look and feel of modal. What you need to do is to use that component.
Also, `Button` is a component as well.